[2023-09-28 06:00:28] INFO: []: [SUCCESS]: Checking Mode = SOCKS


[2023-09-28 06:00:29] INFO: []: [SUCCESS]: Check Passed for: 107.152.98.5:4145
[2023-09-28 06:00:29] INFO: []: [ERROR]:  SOCKSHTTPConnectionPool(host='www.google.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.contrib.socks.SOCKSConnection object at 0x7fc3f01a1c40>: Failed to establish a new connection: 0x04: Host unreachable')) for sock 172.232.185.122:1080
[2023-09-28 06:00:34] INFO: []: [ERROR]:  SOCKSHTTPConnectionPool(host='www.google.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.contrib.socks.SOCKSConnection object at 0x7fc3c001a2e0>, 'Connection to www.google.com timed out. (connect timeout=5)')) for sock 174.138.95.39:36783
[2023-09-28 06:00:37] INFO: []: [SUCCESS]: Check Passed for: 72.195.34.59:4145
[2023-09-28 06:00:39] INFO: []: [SUCCESS]: Check Passed for: 172.104.45.69:1080
[2023-09-28 06:00:44] INFO: []: [ERROR]:  SOCKSHTTPConnectionPool(host='www.google.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.contrib.socks.SOCKSConnection object at 0x7fc3c001a460>, 'Connection to www.google.com timed out. (connect timeout=5)')) for sock 77.68.117.74:53245
[2023-09-28 06:00:44] INFO: []: [ERROR]:  SOCKSHTTPConnectionPool(host='www.google.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.contrib.socks.SOCKSConnection object at 0x7fc3f01a1eb0>: Failed to establish a new connection: [Errno 54] Connection reset by peer')) for sock 184.168.121.153:27697
[2023-09-28 06:00:49] INFO: []: [ERROR]:  SOCKSHTTPConnectionPool(host='www.google.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.contrib.socks.SOCKSConnection object at 0x7fc4001f4e80>, 'Connection to www.google.com timed out. (connect timeout=5)')) for sock 217.172.189.91:14739
[2023-09-28 06:00:50] INFO: []: [ERROR]:  SOCKSHTTPConnectionPool(host='www.google.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.contrib.socks.SOCKSConnection object at 0x7fc3c001b1f0>: Failed to establish a new connection: [Errno 61] Connection refused')) for sock 165.154.233.243:36895
[2023-09-28 06:00:58] INFO: []: [ERROR]:  SOCKSHTTPConnectionPool(host='www.google.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.contrib.socks.SOCKSConnection object at 0x7fc3c001bc10>, 'Connection to www.google.com timed out. (connect timeout=5)')) for sock 45.32.172.135:32940
[2023-09-28 06:01:03] INFO: []: [ERROR]:  SOCKSHTTPConnectionPool(host='www.google.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.contrib.socks.SOCKSConnection object at 0x7fc3f0199f40>, 'Connection to www.google.com timed out. (connect timeout=5)')) for sock 170.187.190.156:1080
[2023-09-28 06:01:08] INFO: []: [ERROR]:  SOCKSHTTPConnectionPool(host='www.google.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.contrib.socks.SOCKSConnection object at 0x7fc3c001a1c0>, 'Connection to www.google.com timed out. (connect timeout=5)')) for sock 115.127.92.50:1088
[2023-09-28 06:01:09] INFO: []: [SUCCESS]: Check Passed for: 199.187.210.54:4145
[2023-09-28 06:01:11] INFO: []: [SUCCESS]: Check Passed for: 98.170.57.231:4145
[2023-09-28 06:01:12] INFO: []: [ERROR]:  SOCKSHTTPConnectionPool(host='www.google.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.contrib.socks.SOCKSConnection object at 0x7fc3c001aee0>: Failed to establish a new connection: [Errno 61] Connection refused')) for sock 193.141.65.48:8975
[2023-09-28 06:01:13] INFO: []: [SUCCESS]: Check Passed for: 5.161.131.42:1823
[2023-09-28 06:01:18] INFO: []: [ERROR]:  SOCKSHTTPConnectionPool(host='www.google.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.contrib.socks.SOCKSConnection object at 0x7fc3f0194280>, 'Connection to www.google.com timed out. (connect timeout=5)')) for sock 139.162.99.100:1080
[2023-09-28 06:01:19] INFO: []: [SUCCESS]: Check Passed for: 65.108.85.95:39053
[2023-09-28 06:01:25] INFO: []: [SUCCESS]: Check Passed for: 138.68.109.12:40766
[2023-09-28 06:01:27] INFO: []: [SUCCESS]: Check Passed for: 98.175.31.195:4145
[2023-09-28 06:01:29] INFO: []: [SUCCESS]: Check Passed for: 184.178.172.28:15294
[2023-09-28 06:01:31] INFO: []: [SUCCESS]: Check Passed for: 68.1.210.189:4145
[2023-09-28 06:01:36] INFO: []: [ERROR]:  SOCKSHTTPConnectionPool(host='www.google.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.contrib.socks.SOCKSConnection object at 0x7fc3c001a670>, 'Connection to www.google.com timed out. (connect timeout=5)')) for sock 5.56.124.176:8192
[2023-09-28 06:01:41] INFO: []: [ERROR]:  SOCKSHTTPConnectionPool(host='www.google.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.contrib.socks.SOCKSConnection object at 0x7fc3f0194730>, 'Connection to www.google.com timed out. (connect timeout=5)')) for sock 122.10.101.145:3333
[2023-09-28 06:01:42] INFO: []: [ERROR]:  SOCKSHTTPConnectionPool(host='www.google.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.contrib.socks.SOCKSConnection object at 0x7fc4001f4b50>: Failed to establish a new connection: 0x04: Host unreachable')) for sock 172.234.38.209:1080
[2023-09-28 06:01:47] INFO: []: [ERROR]:  SOCKSHTTPConnectionPool(host='www.google.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.contrib.socks.SOCKSConnection object at 0x7fc3c001b820>, 'Connection to www.google.com timed out. (connect timeout=5)')) for sock 66.175.238.253:14125
[2023-09-28 06:01:52] INFO: []: [ERROR]:  SOCKSHTTPConnectionPool(host='www.google.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.contrib.socks.SOCKSConnection object at 0x7fc4001f4730>, 'Connection to www.google.com timed out. (connect timeout=5)')) for sock 46.241.57.29:1080
[2023-09-28 06:01:57] INFO: []: [ERROR]:  SOCKSHTTPConnectionPool(host='www.google.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.contrib.socks.SOCKSConnection object at 0x7fc3f01996d0>, 'Connection to www.google.com timed out. (connect timeout=5)')) for sock 188.124.230.43:17662
[2023-09-28 06:01:58] INFO: []: [ERROR]:  SOCKSHTTPConnectionPool(host='www.google.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.contrib.socks.SOCKSConnection object at 0x7fc3c001aee0>: Failed to establish a new connection: 0x04: Host unreachable')) for sock 172.105.176.171:1080
[2023-09-28 06:02:03] INFO: []: [ERROR]:  SOCKSHTTPConnectionPool(host='www.google.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.contrib.socks.SOCKSConnection object at 0x7fc3c001a790>, 'Connection to www.google.com timed out. (connect timeout=5)')) for sock 51.79.165.139:21424
[2023-09-28 06:02:03] INFO: [SUCCESS]: [SUCCESS]: Successfully saved data to working_socks.txt.
[2023-09-28 06:02:03] INFO: [SUCCESS]: [SUCCESS]: Successfully saved data to failed_socks.txt.
